easybuild/easyblocks/i/icc.py

```diff
@@ -44,6 +44,9 @@
 from easybuild.tools.run import run_cmd
 from easybuild.tools.systemtools import get_shared_lib_ext
 
+# location system header files on Debian/Ubuntu that Intel compilers must consider
```

Comment:
only intel, not gcc?

Reply:
this is not supposed to be in here, urgh, will remove...